lms/templates/courses_list.html

```diff
@@ -7,6 +7,13 @@
 
     % if settings.FEATURES.get('COURSES_ARE_BROWSABLE'):
       <section class="courses">
+        <ul class="courses-listing journal-list">
```

Comment:
please add equivalent changes in mitxpro theme whitelabel/journal branch:
https://github.com/edx/edx-themes/blob/whitelabel/journals/edx-platform/mitxpro.mit.edu/lms/templates/courses_list.html#L21